### PR TITLE
[etckeeper] Do not track gitlab runner config, nor docker private key

### PR DESCRIPTION
**Pull request by julien-lecomte**

### CODE REVIEW

File: ansible/roles/etckeeper/defaults/main.yml [hunk 1]
```diff
@@ -161,6 +161,14 @@ etckeeper__default_gitignore:
   - name: 'zfs-zpool-cache'
     ignore: 'zfs/zpool.cache'
 
+  - name: 'gitlab-runner-config'
+    comment: Do not commit GitLab runner private configuration
```
Comment (ypid) [inline ansible/roles/etckeeper/defaults/main.yml]:
```suggestion
    comment: 'Do not commit GitLab runner private configuration'
```

The other items in the list single line comment use single quotes. Just a consistency thing.

File: ansible/roles/etckeeper/defaults/main.yml [hunk 2]
```diff
@@ -161,6 +161,14 @@ etckeeper__default_gitignore:
...
+    ignore: 'gitlab-runner/config.toml'
+
+  - name: 'docker-key'
+    comment: Do not commit Docker private configuration
```
Comment (ypid) [inline ansible/roles/etckeeper/defaults/main.yml]:
```suggestion
    comment: 'Do not commit Docker private configuration'
```

Same.

**Review by ypid — DISMISSED** [review]
LGTM, I only have a style recommendation.

**Review by drybjed — APPROVED** [review]
Looks good, thanks. I'll backport this to the stable releases.

**Pull request merged**